src/data/actions/enrollmentsList.js

```diff
@@ -0,0 +1,56 @@
+import EnterpriseDataApiService from '../services/EnterpriseDataApiService';
+import { getPageOptionsFromUrl } from '../../utils';
+import {
+  PAGINATION_REQUEST,
+  PAGINATION_SUCCESS,
+  PAGINATION_FAILURE,
+} from '../constants/table';
+import NewRelicService from '../services/NewRelicService';
+
+const tableId = 'enrollments';
+
+const searchEnrollmentsListRequest = options => ({
```

Comment:
Why we need this if enrollments were already show on admin portal?

Reply:
we are [dispatching](https://github.com/edx/frontend-app-admin-portal/blob/master/src/data/actions/enterpriseList.js#L43) this action before the fetching search results. we can see this in [reducers](https://github.com/edx/frontend-app-admin-portal/blob/master/src/data/reducers/table.js#L24) that this is setting `loading` state to `true`.